### PR TITLE
Keystone: admins needn't be in the db

### PR DESCRIPTION
**Pull request by zenhack**
...otherwise bootstrapping is hard.

Per our discussion on #615.

//cc @henn, @gsilvis, @knikolla

### CODE REVIEW

**Comment by zenhack** [comment]
Just realized a fairly glaring error in my test; will fix when I get back to my machine.

File: docs/keystone-auth.md [hunk 1]
```diff
@@ -18,7 +18,10 @@ To grant the Openstack project with that UUID access to HaaS.
 
 Note that the plugin recognizes any user with an `admin` role on any
 project as a HaaS administrator, similar to the default policy for core
-Openstack projects.
+Openstack projects. This is true even for projects not that do not exist
+within HaaS; such projects will not be able to own resources (such as
+nodes networks, etc), but may preform admin-only operations (such as
```
Comment (henn) [inline docs/keystone-auth.md]: preform=perform

**Comment by henn** [comment]
Looks good with the anticipated test change.

In addition to a positive test checking that a different project-admin can access stuff, would it be possible to add a negative test too that ensures a regular user with authorization to a non-HIL project can't access anything?

**Comment by zenhack** [comment]
@henn, just fixed the typo and the test, and pushed. Tests pass on my local machine, just waiting on travis.

re: the test you propose, I believe this is what `test_anyone_call_unknown_project` does. Let me know if I've misunderstood what you're asking for.

**Comment by gsilvis** [comment]
Looks good.  +1

**Comment by knikolla** [comment]
+1, merging.

**Pull request merged**